### PR TITLE
Add module for creating RHEL releases

### PR DESCRIPTION
**Pull request by CryptoRodeo**
Fixes: #173

JIRA: CWFCONF-8282

### CODE REVIEW

**Comment by ktdreyer** [comment]
Whoops, I reset the integration test VMs during this run - you should be able to re-push.

**Comment by CryptoRodeo** [comment]
Tested locally, here are the three RHEL releases from the integration tests:
```
[45] pry(main)> RhelRelease.last(3)                
  RhelRelease Load (0.5ms)  SELECT  `rhel_releases`.* FROM `rhel_releases` ORDER BY `rhel_releases`.`id` DESC LIMIT 3
=> [#<RhelRelease:0x00005629fb565f10
  id: 178,
  name: "RHEL-7000",
  description: "RHEL Server 7K",
  exclude_ftp_debuginfo: false,
  version_number: 7000,
  is_ystream: false,
  is_zstream: false,
  is_extended_support: false>,
 #<RhelRelease:0x00005629fb5661b8
  id: 179,
  name: "RHEL-11",
  description: "RHEL Server 11",
  exclude_ftp_debuginfo: true,
  version_number: 11,
  is_ystream: false,
  is_zstream: false,
  is_extended_support: false>,
 #<RhelRelease:0x00005629fb566460
  id: 180,
  name: "RHEL-12.Z.MAIN+EUS",
  description: "RHEL Server 12 Z.MAIN+EUS",
  exclude_ftp_debuginfo: true,
  version_number: 12,
  is_ystream: false,
  is_zstream: true,
  is_extended_support: false>]

```

File: library/errata_tool_rhel_release.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+def get_rhel_release(client, name):
+    """
+    Get a single RHEL release by name.
+
+    :param client: Errata Client
+    :param str name: RHEL release name
+    """
+
+    # Find the RHEL release using the name filter since the api doesn't support
+    # finding by name yet, even though it claims it can.
```
Comment (ktdreyer) [inline library/errata_tool_rhel_release.py:49]: great comment here, thanks

File: library/errata_tool_rhel_release.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+def create_rhel_release(client, params):
+    """
+    Create a new ET rhel release
+
+    :param client: Errata Client
+    :param dict params: ansible module params
+    """
+    # In errata tool, there seems to be 2 formats
+    # for the params when creating or updating a rhel release:
```
Comment (ktdreyer) [inline library/errata_tool_rhel_release.py:76]: Great comment

**Review by ktdreyer — APPROVED** [review]
Great

**Pull request merged**